### PR TITLE
Bug 1816937 - Restore fenix dependency on support-migration

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: repositories.yaml [hunk 1]
```diff
@@ -288,6 +304,7 @@ applications:
       - org.mozilla.components:lib-crash
       - org.mozilla.components:support-sync-telemetry
       - org.mozilla.appservices:logins
+      - org.mozilla.components:support-migration
```
Comment (relud) [inline repositories.yaml:307]: dependencies need to be inclusive of history, or pings from dependencies will be deleted from schemas, which should only be done if we want the ping table deleted.

Comment (travis79) [reply repositories.yaml:307]: Thanks, this is good to know.

**Review by travis79 — APPROVED** [review]
I didn't realize the consequence of removing the dependency, I'll keep an eye out for this in the future. Is this documented somewhere?

**Review by chutten — APPROVED** [review]
LGTM as well.

**Comment by relud** [comment]
> I didn't realize the consequence of removing the dependency, I'll keep an eye out for this in the future. Is this documented somewhere?

I didn't either, but this bug made me notice. I don't even know where this would documented if it were.

**Comment by travis79** [comment]
> > I didn't realize the consequence of removing the dependency, I'll keep an eye out for this in the future. Is this documented somewhere?
> 
> I didn't either, but this bug made me notice. I don't even know where this would documented if it were.

@Dexterp37 was putting together a "run-book" for adding applications for ingestion, perhaps that would be a good place to make note of this?

**Comment by Dexterp37** [comment]
@relud @travis79 I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1817246

**Pull request merged**